src/bennu/executables/bennu-simulink-provider/main.cpp

```diff
@@ -284,7 +286,8 @@ class BennuSimulinkProvider : public Provider
         {
             publishData();
             std::cout << std::flush;
-            std::this_thread::sleep_for(std::chrono::seconds(1));
+            // std::this_thread::sleep_for(std::chrono::seconds(1));
+            std::this_thread::sleep_for(durationToDuration(this->publishRate_setting));
```

Comment:
where is durationToDuration defined? Did you forget to include in the PR?

Reply:
Included function declaration above BennuSimulinkProvider class